src/components/policy/policy_external/src/policy_manager_impl.cc

```diff
@@ -986,10 +984,6 @@ void PolicyManagerImpl::SetUserConsentForApp(
     const PermissionConsent& permissions, const NotificationMode mode) {
   LOG4CXX_AUTO_TRACE(logger_);
 
-  if (permissions.group_permissions.empty()) {
-    LOG4CXX_DEBUG(logger_, "Permissions list is empty, skipping update.");
-    return;
-  }
```

Comment:
@LuxoftAKutsan Why this code was removed?

Reply:
We should be able to apply user consent even is groups are empty